secureli/utilities/usage_stats.py

```diff
@@ -15,3 +18,22 @@ def convert_failures_to_failure_count(failure_list: list[Failure]):
 
     failure_count_list = Counter(list_of_failure_ids)
     return failure_count_list
+
+
+def post_log(log_data: str):
+    """
+    Send a log through http post
+    :param log_data: a string to be sent to backend instrumentation
+    """
+
+    API_ENDPOINT = os.getenv("API_ENDPOINT")
+    API_KEY = os.getenv("API_KEY")
+
+    if not API_ENDPOINT or not API_KEY:
```

Comment:
I would not require an API_KEY, but include it in the header if they actually provide it. Just in case someone would want to set up their own endpoint without a key.

Reply:
Nevermind.  That doesn't sound very secure by default.  We should require some sort of key to post usage stats.